modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -9678,6 +9678,8 @@ type Observation {
   configurationRequests: [ConfigurationRequest!]!
 
   workflow: ObservationWorkflow!
+
+  calculatedWorkflow: CalculatedObservationWorkflow
```

Comment:
I don't know whether we need the original workflow computation in the API?  I wonder if we can get away with just calling it `workflow` and returning a `CalculatedObservationWorkflow`.

Reply:
That seems fine to me. Same with removing the top-level query.